processing/app/main.py

```diff
@@ -1,12 +1,34 @@
 from fastapi import FastAPI
-from .models import Item
+from sqlmodel import Session, select
+
+from app.database import create_db_and_tables, engine
+from .models import AnalysisJob, Item
+
 
 app = FastAPI()
 
+@app.on_event("startup")
+def on_startup():
+    create_db_and_tables() # Create tables if they don't exist
+    with Session(engine) as session:
+            # Check if the item already exists to avoid duplicate inserts
+            exists = session.exec(select(AnalysisJob).where(AnalysisJob.survey_id == "1")).first()
+            if not exists:
+                item = AnalysisJob(survey_id="1")
+                session.add(item)
+                session.commit()
+
 @app.get("/")
 def read_root():
     return {"message": "Hello, World!"}
 
-@app.post("/items/")
+@app.post("/items")
 def create_item(item: Item):
     return {"item_received": item}
+
+@app.get("/jobs", response_model=list[AnalysisJob])
+def read_jobs():
```

Comment:
As mentioned in a comment [above](https://github.com/opendexcom/formul.ai/pull/41/files#r2070388766) inject engine via FastAPI Dependency.